app/models/event.rb

```diff
@@ -2,53 +2,83 @@ class Event < ActiveRecord::Base
   before_save :calc_duration
 
   attr_accessible :title, :description, :category, :course_id, :duration, :start_time, :end_time, :instructor, :location, :status, :timecard_ids, :person_ids, :comments
- 
-  def end_date_cannot_be_before_start
-    if ((!end_time.blank?) and (!start_time.blank?)) and end_time < start_time
-      errors.add(:end_time, "must be after the start, unless you are the Doctor")
-    end
-  end
+
   def end_date_cannot_be_before_start
```

Comment:
For some reason, I was thinking it had to be defined before it could be called. Looking [at Ruby Guides](http://guides.rubyonrails.org/active_record_validations.html#custom-methods) shows that is not the case.